java/src/jmri/jmrix/can/cbus/swing/cbusslotmonitor/CbusSlotMonitorPane.java

```diff
@@ -45,6 +67,20 @@
     
     protected CbusSlotMonitorDataModel slotModel=null;
     protected JTable slotTable=null;
+    protected final XTableColumnModel tcm = new XTableColumnModel();
+
+    private JMenu cabsigMenu = new JMenu("CabData Options");
+    private JMenu cabsigSpeedMenu = new JMenu(Bundle.getMessage("Speed"));
+    private JMenu colMenu = new JMenu((Bundle.getMessage("SessCol")));
+    private JMenu cabSigColMenu = new JMenu(("CabData Columns"));
+    private JMenu debugMenu = new JMenu(Bundle.getMessage("CBUS_DBG1"));
+    
+    // private JMenu cancmdMenu = new JMenu("CANCMD Setup");
+    protected static JRadioButtonMenuItem lowdebugitem = new JRadioButtonMenuItem(Bundle.getMessage("Low"));
```

Comment:
Should these be static?  (Really a question, I haven't been able to follow the logic to know yes or no)

Reply:
"static" on a member means shared across all objects.  I can imagine having more than one SlotMonitor open; we certainly don't prevent that.  So better to allow more than one by not having them static.

In that case, an internal class that needs to reference them can't itself be static.  But that's not a problem, most internal classes aren't.

So I recommend removing the method and internal-class static modifiers.

Reply:
Thanks for the feedback Bob, will have another look at this,
Steve.

Reply:
Hi Bob,
Have made a lot less static,
tested with multiple instances all ok.